valhalla/midgard/gridded_data.h

```diff
@@ -98,7 +98,7 @@ template <std::size_t dimensions_t> class GriddedData : public Tiles<PointLL> {
     typename PointLL::first_type s[5]; // Values at the tile corners and center
```

Comment:
why is the code from https://github.com/valhalla/valhalla/pull/2932 in this PR as well? something seems messed up here?

Reply:
Originally i found 2 issues at once(polygon area bug and orientation bug) so I decided to make a hierarchy `master <- 2927 <- 2932` for some reason 🤦(probably because of the unit tests)  Now I think It brought more mess than clearness. Anyway, I had to fix unit tests so it didn't bring any profit
Would you like to split them @kevinkreiser ? (I guess I do, it'll just take a little more time and a few more approves)

Reply:
oh no, no problem at all, i just missed a very important detail:

![image](https://user-images.githubusercontent.com/697548/113886123-6d4f5d00-978e-11eb-9f75-98734a179e84.png)

the previous one was merged into this branch not master. i completely missed that. this is fine!